L-I/0006 Repeat a String (L-I)/README.md

```diff
@@ -0,0 +1,33 @@
+# 0006 Repate a String Num times
+
+repeat  a given string `str` for `num` of times, and return empty in case the num is zero or negative number. 
+
+#### Note: you are not allowd to use `.repeat()` method.
+
+## For testing:
+
+- run 
+```
+npm i
+```
+to install mocha library, then run
+```
+npm test
+```
+
+## Solution:
+
+>- JavaScript.
+
+```
+const repeatStringNumTimes = (str, num) => {
+  if (num < 1) {
+    return "";
+  }
+  // here I used recursion technique to solve the problem
+  return str + repeatStringNumTimes(str, num - 1);
+};
+```
+
+[click](https://www.khanacademy.org/computing/computer-science/algorithms/recursive-algorithms/a/recursion) To lern more about recursion method 
+
```

Comment:
please follow the guidelines on how you can make a perfect readme file [link](https://github.com/devvsakib/javascript-problem-solving/blob/main/L-B/0002%20ContainString%20(%20L-B%20))/README.md)